projects/dotcms-field-elements/src/components/dot-date-time/dot-date-time.e2e.ts

```diff
@@ -1,140 +1,446 @@
+            });
+
+            it('should pass correctly date value and empty time', async () => {
+                element.setProperty('value', '2019-01-01');
+                await page.waitForChanges();
+                expect(await dateInput.getProperty('value')).toBe('2019-01-01');
+                expect(await timeInput.getProperty('value')).toBeNull();
+            });
 
-            expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
-                name: 'dt01',
-                status: {
-                    dotPristine: true,
-                    dotTouched: true,
-                    dotValid: true
-                }
+            it('should pass correctly time value and empty date', async () => {
+                element.setProperty('value', '10:10:01');
+                await page.waitForChanges();
+                expect(await dateInput.getProperty('value')).toBeNull();
+                expect(await timeInput.getProperty('value')).toBe('10:10:01');
+            });
+
+            it('should set empty value if format is not correct ', async () => {
+                element.setProperty('value', { test: 'hi' });
+                await page.waitForChanges();
+                expect(await dateInput.getProperty('value')).toBeNull();
+                expect(await timeInput.getProperty('value')).toBeNull();
             });
         });
 
-        it('should send status and value change and stop dot-input-calendar events', async () => {
-            const evt_statusChange = await page.spyOnEvent('_statusChange');
-            const evt_valueChange = await page.spyOnEvent('_valueChange');
-            await dateInput.press('2');
-            await timeInput.press('5');
-            await page.waitForChanges();
-            expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
-                name: 'dt01',
-                status: {
-                    dotPristine: false,
-                    dotTouched: true,
-                    dotValid: true
-                }
+        describe('name', () => {
+            it('should pass correctly to dot-input-calendar', async () => {
+                element.setProperty('name', 'test');
+                await page.waitForChanges();
+                expect(dateInput.getAttribute('name')).toBe('test-date');
+                expect(timeInput.getAttribute('name')).toBe('test-time');
             });
-            expect(spyValueChange).toHaveReceivedEventDetail({
-                name: 'dt01',
-                value: '2019-02-20 05:10:00'
+
+            it('should set name prop in dot-label', async () => {
+                element.setProperty('name', 'test');
+                await page.waitForChanges();
+                const label = await dotTestUtil.getDotLabel(page);
+                expect(label.getAttribute('name')).toBe('test');
+            });
+
+            it('should render default value', () => {
+                expect(dateInput.getAttribute('name')).toBe('-date');
```

Comment:
Should we just don't pass name if we don't have it?

Reply:
no, because is the way to identify the events that each `dot-input-calendar` emits